lib/ansible/modules/system/iptables.py

```diff
@@ -503,6 +529,10 @@ def main():
             uid_owner=dict(required=False, default=None, type='str'),
             reject_with=dict(required=False, default=None, type='str'),
             icmp_type=dict(required=False, default=None, type='str'),
+            syn=dict(
+                required=False,
```

Comment:
Flag `required=False` is not needed, the is implied if it is not `required=True`